tests/migrations/test_commands.py

```diff
@@ -727,17 +729,34 @@ def test_makemigrations_interactive_accept(self):
             out = six.StringIO()
             try:
                 with self.temporary_migration_module(module="migrations.test_migrations_conflict") as migration_dir:
-                    call_command("makemigrations", "migrations", merge=True, interactive=True, stdout=out)
+                    call_command("makemigrations", "migrations", name="merge", merge=True,
+                                 interactive=True, stdout=out)
                     merge_file = os.path.join(migration_dir, '0003_merge.py')
                     self.assertTrue(os.path.exists(merge_file))
             except CommandError:
                 self.fail("Makemigrations failed while running interactive questioner")
             self.assertIn("Created new merge migration", force_text(out.getvalue()))
 
+    def test_makemigrations_default_merge_name(self):
+        test_date = datetime.datetime(2016, 1, 2, 3, 4)
+        test_date_mock = mock.patch('django.utils.timezone.now', mock.Mock(return_value=test_date))
+        with mock.patch('django.db.migrations.questioner.input', mock.Mock(return_value='y')), test_date_mock:
+            out = six.StringIO()
+
+            try:
```

Comment:
Yes, please rebase the branch and remove the try/fail pattern as done in 6729b96d8a15048b2295c916c5b881a59d9417a0. If you're unfamiliar with the process you might find https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/working-with-git/#rebasing-branches helpful.